desc/plotting.py

```diff
@@ -2234,8 +2226,10 @@ def plot_boozer_surface(
     eq,
     grid_compute=None,
     grid_plot=None,
+    rho=1,
```

Comment:
Since we're adding this, does it make sense to move the `grid_compute` and `grid_plot` to be keyword only args? And possibly add similar positional args for ntheta/nzeta? Several of the other plotting functions are similar.